src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -301,13 +302,21 @@ public void verifyGptIframe(String adUnit, String slotName, String src) {
    * @param src         the source of an ad, for example gpt, remnant or empty
    */
   public String buildGptIframeId(int dfpClientId, String adUnit, String slotName, String... src) {
-    return Joiner.on("/").skipNulls().join(
-        "google_ads_iframe_",
-        String.valueOf(dfpClientId),
-        adUnit,
-        src.length > 0 ? src[0] : null,
-        slotName + "_0"
-    );
+    if (slotName == "BOTTOM_LEADERBOARD"){
```

Comment:
I believe it's due to the maladie project and therefore it's fine to keep such code change for a while IMHO. Do we have a follow-up ticket to clean it after we finished maladie and/or MEGA? What do you think of placing a comment before this change with a ticket number, something like
```
// TODO remove the if-statement after ADEN-xxxx is done